packs/data/pathfinder-bestiary-3.db/kirin.json

```
@@ -1284,7 +1401,37 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "rearing-thrust",
+                        "toggleable": true
+                    },
+                    {
+                        "diceNumber": 1,
+                        "dieSize": "d8",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "rearing-thrust"
+                        ],
+                        "selector": "horn-damage"
+                    },
+                    {
+                        "key": "Note",
+                        "outcome": [
+                            "success",
+                            "criticalSuccess"
+                        ],
+                        "predicate": [
+                            "rearing-thrust"
+                        ],
+                        "selector": "horn-attack",
+                        "text": "@Localize[PF2E.AttackEffects.KnockProne]",
```

```suggestion
                        "text": "PF2E.AttackEffects.KnockProne",
```
We don't need to call to localize these, that is done during the processing of the note anyways.